packages/gatsby/src/internal-plugins/query-runner/query-watcher.js

```diff
@@ -189,7 +190,7 @@ const watch = rootDir => {
   }, 100)
 
   watcher = chokidar
-    .watch(path.join(rootDir, `/src/**/*.{js,jsx,ts,tsx}`))
+    .watch(slash(path.join(rootDir, `/src/**/*.{js,jsx,ts,tsx}`)))
```

Comment:
`slash` to the rescue again

Reply:
There is also other issue with watching files, but I will create separate PR for it

Reply:
Is this PR good to merge?